src/Forms/Checkbox.js

```diff
@@ -143,8 +143,15 @@ Please ensure you are either using a visible \`FormLabel\` or an \`aria-label\`
     labelProps: PropTypes.object,
     /** Sets the `name` for the checkbox input */
     name: PropTypes.string,
-    /** Additional classes to apply to validation popover */
-    validationClassName: PropTypes.string,
```

Comment:
Should moving this prop to `validationOverlayProps.className` be listed as a breaking change as well?

Reply:
100%! Good catch